testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -49,11 +50,15 @@ use std::{
 use tokio::{task::JoinHandle, time};
 
 const MAX_TXN_BATCH_SIZE: usize = 100; // Max transactions per account in mempool
+const MAX_CHILD_VASP_NUM: usize = 256;
```

Comment:
There is same constant in `VASP.move`, named MAX_CHILD_ACCOUNTS

We can't reuse that constant, but can we cross-comment here and in the move file that those constants needs to be updated at the same time?